tests/users/test_signup.py

```diff
@@ -22,7 +43,8 @@ def test_signup_user_newsletter_checked(client):
     )
     assert resp.status_code == 302
     user = User.objects.get()
-    assert user.get_newsletters
+    assert user.notification_settings.email_newsletter
+    assert not user.notification_settings.notify_creator
```

Comment:
why here the notification is false for notify_creator and in the previous test is true? I don't see a change in the payload

Reply:
`"get_notifications": "on",` is not set in the first test.